src/jdaTemplate/app/build.gradle

```
@@ -0,0 +1,19 @@
+plugins {
+    // Apply the application plugin to add support for building a CLI application in Java.
+    id 'application'
+}
+
+repositories {
+    // Use Maven Central for resolving dependencies.
+    mavenCentral()
+}
+
+dependencies {
+    // Discord API
+    implementation("net.dv8tion:JDA:5.0.0-alpha.4")
```

JDA uses SLF4J in order to log messages. Without any logger implementation, it would hide all logging. An example of such a logger would be `slf4j-simple`.
```suggestion
    implementation("net.dv8tion:JDA:5.0.0-alpha.4")
    runtimeOnly("org.slf4j:slf4j-simple:1.7.36")
```
See [this](https://github.com/DV8FromTheWorld/JDA#logging-framework---slf4j) for details.